dev-libs/qtcompress/qtcompress-0_p20180513.ebuild

```
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils toolchain-funcs virtualx
+
+DESCRIPTION="A public API for QZip in an easy to use module"
+
+HOMEPAGE="https://github.com/nezticle/qtcompress"
+MY_COMMIT="23f8831826cd72aedf99fc3699148b6c994fd677"
+SRC_URI="https://github.com/nezticle/qtcompress/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( LGPL-2.1 GPL-3 ) FDL-1.3"
+SLOT="0/5.11.0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
```

You could separate these with an empty line next time touching this.

---

Separate them from each other, or from something else?

---

Yes, I still find it too clumped together, makes it more readable to add one newline between.
```suggestion
LICENSE="|| ( LGPL-2.1 GPL-3 ) FDL-1.3"
SLOT="0/5.11.0"
KEYWORDS="~amd64 ~x86"
IUSE="test"

RESTRICT="!test? ( test )"
S="${WORKDIR}/qtcompress-${MY_COMMIT}"
```
Although I'd personally even separate `S` from this group, but style things.